index.html

```diff
@@ -1767,7 +1769,7 @@ <h2>The <dfn>record type</dfn> string</h2>
     <tr>
       <td nowrap><dfn>"`smart-poster`"</dfn></td>
       <td><i>unused</i></td>
-      <td>{{NDEFMessage}}</td>
```

Comment:
Nit: there's another occurrence of `NDEFMessage` in that column.

Reply:
Thanks for catching!